watcher.py

```
@@ -1,42 +1,47 @@
+
+class ChangeHandler(FileSystemEventHandler):
+    def on_any_event(self, event):
+        if event.is_directory:
+            return
+        # Trigger content generation and posting for any file in the folder
+        print(f"Change detected in {event.src_path}, generating post...")
+        # Example: call your FastAPI endpoint
+        try:
+            prompt = f"Change detected in {event.src_path}"
+            # Generate content
+            gen_resp = requests.post("http://127.0.0.1:8000/generate-content/", json={"prompt": prompt})
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)